Add clarification about Python distribution (#3367)
astronomer/docs: astro/runtime-image-architecture.md

quay.io/astronomer/astro-runtime:<runtime-version>-python-<python-version>
```

> Each Python distribution also includes an alternative `quay.io/astronomer/astro-runtime:<version>-base` distribution if you need to further customize the image.

Author: ```suggestion
Each Python distribution also has an alternative `quay.io/astronomer/astro-runtime:<version>-base` distribution that you can use to further customize the image.
```
